test-suite/rust-runtime-api/build.rs

```diff
@@ -14,9 +14,10 @@
 
 use exonum_build::ProtobufGenerator;
 
+// API Tests.
```

Comment:
Nit: This comment is misleading, it's not "API tests", it's a main function of a build script :slightly_smiling_face: